sandpack-react/src/components/CodeEditor/styles.ts

```diff
@@ -27,6 +27,9 @@ export const editorClassName = css({
   [`.${placeholderClassName}`]: {
     padding: "$space$4 0",
   },
+  "&:focus-within": {
+    outline: "$colors$accent auto 1px",
```

Comment:
@shrilakshmishastry, this can be implemented on your end, meaning inside the application that implements Sandpack, instead of adding a default style that might not be wanted by everyone. For a basic and generic experience, the editor already provides the necessary hint when it's focused, so I don't think adding such an opinionated style will be a good idea now. 

For example, this can be added to your application:
```css
.sp-code-editor:focus-within  {
  outline: 1px auto red;
}
```

Reply:
hmm, got it 👍  make sense, I will revert this

Reply:
reverted [chore: remove focus-within added in codeEditor/styles](https://github.com/codesandbox/sandpack/pull/1184/commits/e26006e05ceb5fd10f9dca6157990f48b93a3911)

Reply:
![image](https://github.com/user-attachments/assets/14a2faea-b82a-45cc-8cf6-8b16fe02d233)